test/Interop/Cxx/constructor/copy-basic.swift

```diff
@@ -0,0 +1,55 @@
+// RUN: %target-run-simple-swift(-I %S/Inputs -Xfrontend -enable-cxx-interop)
```

Comment:
A note on test directory organization: I don't think we need a separate directory `constructor` -- that seems a bit fine-grained. I'd suggest putting this in `test/Interop/Cxx/class`, which is where the constructor tests in #30630 also live.

The name of the tests can then become something like `copy-constructor` with appropriate suffixes (also for the C++ input file).

Reply:
WDYT about `test/Interop/Cxx/value-witness-table` for the value witness table bits, and `test/Interop/Cxx/class` for changes in the imported Swift API (if we decide to make copy constructors visible in the API)?

Reply:
That works too.

Reply:
Done.